packages/plugin-infra/src/web3/useSocialAddressListAll.ts

```diff
@@ -26,7 +26,7 @@ export function useSocialAddressListAll(
 
     return useAsyncRetry(async () => {
         const userId = identity?.identifier?.userId
-        if (!userId || userId === '$unknown' || !identity?.publicKey) return EMPTY_LIST
```

Comment:
@Randolph314 Why check `identity.publicKey` here? It could always be `undefined`